src/afl-analyze.c

```diff
@@ -989,6 +993,14 @@ int main(int argc, char **argv_orig, char **envp) {
 
         break;
 
+      case 'A':                                           /* CoreSight mode */
+
+        if (cs_mode) { FATAL("Multiple -A options not supported"); }
```

Comment:
Two questions come to mind:
- should the flag be available on non aarch64 compile targets? What about non-linux?
- do we need a new flag all all?

Reply:
I added the platform checks by c2feee4.